src/data/constants.ts

```diff
@@ -1,6 +1,16 @@
 export const VSCODE_PROJECT_CONFIG = {
   project: '<your-key-here>',
   sonarURL: '<your-sonar-url>',
+  auth:
```

Comment:
Need to add support to provide `token` instead of `username` and `password` too. So maybe will add some documentation to let users know about it.

Reply:
Hi

 I done two way of authentication. As on my site part of persons use token or username and password. I will clean up the code and create PR today or tomorrow

Reply:
https://github.com/adisreyaj/vscode-sonarqube-status/pull/30

I've added token based auth too. Would appreciate if you can try it out locally once.